### PR TITLE
Add routing to website

### PR DESCRIPTION
**Pull request by eyesaacli**
Testing Steps
------------------------------------------------------------------------
1. yarn && yarn build && yarn start
2. navigate to localhost:3000
3. click 'go to rider signup' button to navigate to rider signup form
4. ensure that form is at localhost:3000/rides/rider/signup

Resources
------------------------------------------------------------------------
- react-router: https://reactrouter.com

### CODE REVIEW

**Comment by eriadnus** [comment]
General question, did you do the thank you page-ish here too? That's not really in the scope of this task.

File: src/Components/Rides/RiderSignup/RiderSignup.tsx [hunk 1]
```diff
@@ -2,36 +2,71 @@
...
  */
 
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { Button, Col, Container, Form, Row } from 'react-bootstrap';
 import { Turnarounds } from '../../../Constants/RidesConstants';
+import { useNavigate } from 'react-router-dom';
 
 import './RiderSignup.scss';
-import 'css/common/forms.scss'
```
Comment (eriadnus) [inline src/Components/Rides/RiderSignup/RiderSignup.tsx:10]: Nice catch!

File: src/Components/Rides/RiderSignup/RiderSignup.tsx [hunk 2]
```diff
@@ -2,36 +2,71 @@
...
+                  {' '}
+                  Name *{' '}
+                </Form.Label>
                 <Form.Control placeholder={'Enter your name'} />
               </Form.Group>
 
               <Form.Group className={'mb-3'} controlId={'riderEmail'}>
-                <Form.Label className={'signup-form-label'}> Email * </Form.Label>
+                <Form.Label className={'signup-form-label'}>
+                  {' '}
```
Comment (eriadnus) [inline src/Components/Rides/RiderSignup/RiderSignup.tsx:59]: Why is this stuff added? Does it look weird on your computer?

File: src/Components/Rides/RiderSignup/RiderSignup.tsx [hunk 3]
```diff
@@ -2,36 +2,71 @@
...
+    return <div>Thank You!</div>;
+  }
 
   return (
     <Container>
       <Col className={'mx-auto text-center'} lg={'8'}>
         <span className={'signup-form-title'}>Sign up for a ride to Church!</span>
 
-        <Form className={'text-center'}>
+        <Form className={'text-center'} onSubmit={handleSubmit}>
```
Comment (eriadnus) [inline src/Components/Rides/RiderSignup/RiderSignup.tsx:46]: Form submit stuff isn't really in the scope of this assignment. We'll have a separate task to handle the redirect after users sign up (this is probably after the Thank You page gets finished.

File: src/Components/Rides/Rides.tsx [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```
Comment (eriadnus) [inline src/Components/Rides/Rides.tsx:1]: Beyond scope of this task

File: src/Components/Rides/Rides.tsx [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+
+export const Rides = () => {
+  return <div>Rider or Driver?</div>;
```
Comment (eriadnus) [inline src/Components/Rides/Rides.tsx:4]: ^

File: src/index.tsx [hunk 6]
```diff
@@ -2,12 +2,23 @@ import React from 'react';
...
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { Rides } from './Components/Rides/Rides';
+import { RiderSignup } from './Components/Rides/RiderSignup/RiderSignup';
+import { DriverSignup } from './Components/Rides/DriverSignup/DriverSignup';
+import { Landing } from './Components/Landing/Landing';
 
 ReactDOM.render(
   <React.StrictMode>
-    <RiderSignup />
+    <BrowserRouter>
```
Comment (eriadnus) [inline src/index.tsx:14]: Don't worry about creating all the other routes - we'll handle it piece-by-piece (i.e. when we create the pages, we'll add the routes in).

**Review by eriadnus — CHANGES_REQUESTED** [review]
Reduce the scope of this task a bit. We'll take things piece-by-piece. For the routing, you only need to worry about adding in the '/rides/rider/signup' (and implicitly the landing page '/'). Not sure about the formatting changes to the form - maybe this is something we need to talk about?

File: src/Components/Rides/RiderSignup/RiderSignup.tsx (hunk 2 above)
Comment (eyesaacli) [reply src/Components/Rides/RiderSignup/RiderSignup.tsx:59]: my formatter automatically added that i'll have to look into the settings of my formatter to change it

File: src/Components/Rides/RiderSignup/RiderSignup.tsx (hunk 3 above)
Comment (eyesaacli) [reply src/Components/Rides/RiderSignup/RiderSignup.tsx:46]: ohh got it. i thought i had to do everything on the trellis task

**Pull request merged**